[docs] POST /_aliases remove_index action only works on concrete indices (#64616)
elastic/elasticsearch: docs/reference/indices/aliases.asciidoc

Removes an alias from an index.

`remove_index`::

> Deletes a concrete index. Similar to the <<indices-delete-index, delete index API>>, but if the provided index is an alias, the action will fail.

Reviewer: ```suggestion
Deletes a concrete index, similar to the <<indices-delete-index, delete index
API>>. Attempts to remove an index alias will fail.
```
